pkg/kubernetes/manifest/manifest.go

```diff
@@ -9,9 +9,19 @@ import (
 	"github.com/Masterminds/sprig/v3"
 	"github.com/pkg/errors"
 	"github.com/stretchr/objx"
-	yaml "gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v3"
```

Comment:
That's going to be a blocker for us 😞. v3 has major formatting differences from v2, and there's no way to configure it (see https://github.com/grafana/tanka/issues/607#issuecomment-973683064). For anyone using GitOps, this upgrade is very troublesome because it means that most Kubernetes resources will be redeployed

Reply:
Damn, didn't think about this, thought this had been resolved already. Do you consider switching to the fork the kustomize folks are using right now? With the v2 API, we cannot have such functionalities

Reply:
ping :)

Reply:
If we can maintain the same indentations as we currently have, and it supports new features such as this, then I think it'd be a good option

Reply:
So - the short version of this is that we cannot, I'll check if I can come up with another way, maybe a feature flag like `--use-yaml-v3`?